components/NotificationBar.js

```diff
@@ -1,12 +1,63 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import styled from 'styled-components';
 
 import AcceptRejectButtons from './host-dashboard/AcceptRejectButtons';
+import Container from './Container';
 import { Flex } from './Grid';
+import { H1, P } from './Text';
 
 const logo = '/static/images/opencollective-icon.svg';
 
+const NotificationBarContainer = styled.div`
+    .oc-message .logo {
+      margin: 10px;
+    }
+
+    .error .oc-message {
+      position: fixed;className="oc-message"
```

Comment:
Seems there's a problem here

Reply:
Thanks. That's some unintended pasting on my part. Just removed it. 👍🏽